lib/actions/pauseall.js

```diff
@@ -58,8 +58,27 @@ function doResumeAll(system) {
   return Promise.all(promises);
 }
 
+function anyPlayerOn(player, values) {
+  logger.debug("is any player on");
+
+  if (values[0] && values[0] > 0) {
+    setTimeout(function () {
+      doStateOn(player.system);
+    }, values[0] * 1000 * 60);
+    return Promise.resolve();
+  }
+
+  return doAnyPlayerOn(player.system);
+}
+
+function doAnyPlayerOn(system) {
+  const promises = system.zones.some(zone => zone.coordinator.state.playbackState === 'PLAYING');
+  return {promises};
+}
+
 
 module.exports = function (api) {
   api.registerAction('pauseall', pauseAll);
   api.registerAction('resumeall', resumeAll);
-}
+  api.registerAction('anyplayeron', anyPlayerOn);
```

Comment:
This should probably be in it's own action file, since it doesn't really have anything to do with play/pause all functionality.